src/Sentry.Profiling/SampleProfilerSession.cs

```diff
@@ -22,21 +21,21 @@ private SampleProfilerSession(SentryStopwatch stopwatch, EventPipeSession sessio
     {
         _session = session;
         _logger = logger;
-        _eventSource = eventSource;
-        _sampleEventParser = new SampleProfilerTraceEventParser(_eventSource);
+        EventSource = eventSource;
+        _sampleEventParser = new SampleProfilerTraceEventParser(EventSource);
         _stopwatch = stopwatch;
     }
 
     // Exposed only for benchmarks.
     internal static EventPipeProvider[] Providers = new[]
     {
-        // Note: all events we need issued by "DotNETRuntime" provider are at "EventLevel.Informational"
-        // see https://learn.microsoft.com/en-us/dotnet/fundamentals/diagnostics/runtime-events
-        // TODO replace Keywords.Default with a subset. Currently it is:
-        //   Default = GC | Type | GCHeapSurvivalAndMovement | Binder | Loader | Jit | NGen | SupressNGen
-        //                | StopEnumeration | Security | AppDomainResourceManagement | Exception | Threading | Contention | Stack | JittedMethodILToNativeMap
-        //                | ThreadTransfer | GCHeapAndTypeNames | Codesymbols | Compilation,
-        new EventPipeProvider(ClrTraceEventParser.ProviderName, EventLevel.Informational, (long) ClrTraceEventParser.Keywords.Default),
+        new EventPipeProvider(ClrTraceEventParser.ProviderName, EventLevel.Verbose, (long) (
```

Comment:
`EventLevel.Verbose` is the actual fix here. The rest of the change is just an improvement so we don't receive things we don't need.